custom_rpc.json

```diff
@@ -5,96 +5,103 @@
         "params": [{
             "name": "blockHash",
             "type": "Hash",
-            "isOptional": "true"
+            "isOptional": true
         },
         {
             "name": "did",
-            "type": "H256",
-            "isOptional": "false"
+            "type": "IdentityId",
+            "isOptional": false
         },
         {
             "name": "buffer_time",
             "type": "u64",
-            "isOptional": "true"
-        }]
+            "isOptional": true
+        }],
+        "type": "Result<CddStatus>"
```

Comment:
These types seem redundant. The polkadot API is getting things like `{"Ok":{"Result":{"Ok":"0x437319f49661c6c4100bcc99b5c2a8031d360fbd2a726dc23d00529e5d977fd8","Err":"0x"}}}`